bin/metrics-regress

```diff
@@ -1,4 +1,33 @@
 #!/usr/bin/python3
```

Comment:
Minor note.  Usually like to do /usr/bin/env python3 just in case an install is using a custom Python (or a virtual env).
I know we aren't supposed to manually but provides a little resilience to Python environment.